Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:12-alpine3.10
+
```

Comment:
I am curious on the use case of having this in a Docker image. Nothing wrong with building it, just curious what the intention of it is. Are you using this in a side car container?